src/components/wallet/open.js

```diff
@@ -94,52 +97,62 @@ class WalletForm extends React.Component {
         reader.onload = (e) => {
           try {
               const pw = Wallet.walletRequirePass(e.target.result);
-              this.setState({ showRequirePass: pw, file: e.target.result });
+              this.setState({ showRequirePass: pw, file: e.target.result,showAccessButton: true });
           } catch (e) {
               console.error(e)
           }
         };
-  }
+    }
+  showIntroText(){
+      this.setState({ showIntro: true });
+    }
 
   render() {
     return (
       <Grid>
         <Row>
-          <Col sm={12} md={4}>
+          <Col sm={12} md={4} >
             <h4>Select the format of your private key.</h4>
             <Form>
-              <FormGroup>
+              <FormGroup onLoad={this.showIntroText}>
                 <Radio name="keyFormat" value="file" onChange={this.handleFormat}>
                   JSON or Keystore File</Radio>
                 <Radio name="keyFormat" value="text" onChange={this.handleFormat}>
                   Plain Text Private Key</Radio>
               </FormGroup>
             </Form>
           </Col>
-          {this.state.showTextKey && <Col sm={12} md={4}>
+          {this.state.showIntro && <Col sm={12} md={4} lg={4}>
+          <Well>
+          <p>
+          Please provide the private key for the address that you would like to use to create your ERC 223 tokens. The address you choose requires a minimum amount of ETC or ETH to pay for the Gas (Transaction Fee) necessary to execute the smart contract which creates the tokens you wish to create. We recommend having at least .1 ETC or .1 ETH in order to execute the smart contract and be able to move your tokens when you wish.
```

Comment:
let's leave any mention of ETH out until we actually support it (don't want ppl losing ETH trying to create tokens here)